spec/requests/api/arbitration_profile_spec.rb

```diff
@@ -30,7 +31,6 @@
       expect do
         run_post(arbitration_profiles_url, request_body)
       end.to change(ArbitrationProfile, :count).by(1)
-      expect(response).to have_http_status(:ok)
```

Comment:
only because I saw that I didn't have that in similar places - although in retrospect, I should've done the opposite to fix consistency